### ISSUE TITLE
Cannot build caddy v2 with the nginx-adapter!

### ISSUE THREAD

**Issue by artknight** [caddyserver/nginx-adapter#23]
<img width="1602" alt="Screen Shot 2022-01-20 at 2 39 18 PM" src="https://user-images.githubusercontent.com/157762/150410241-f77314f0-03cf-4a2c-b6e6-91b848dd6305.png">

**Comment by mholt** [comment]
The build error is:

```
nginx-adapter@v0.0.3/nginxadapter.go:66:17: undefined: caddyconfig.JSONIndent
```

Which is weird since L66 doesn't have that.

**Comment by artknight** [comment]
@mholt why is this ticket closed? Is the issue resolved?

**Comment by artknight** [comment]
@mholt oops, my apologies.. I saw the `closed` tag and assumed it was for this issue.. sorry

**Comment by mohammed90** [comment]
> The build error is:
> 
> ```
> nginx-adapter@v0.0.3/nginxadapter.go:66:17: undefined: caddyconfig.JSONIndent
> ```
> 
> Which is weird since L66 doesn't have that.

v0.0.3 used it and depended on it, but the usage was removed in commit 50635ac4cf58f1a0f7d6f8a8ae059c3681fc4088 when the func was removed from caddy. I never released new tag with that particular fix (see #21). I've released v0.0.4 now which includes the fix.

**Comment by mholt** [comment]
Sorry about the break, heh. Thanks Mohammed
